From 75c4b4e32d95acc58812a6cad9d1a63771e9d53a Mon Sep 17 00:00:00 2001
From: Moritz Schulte <mo@g10code.com>
Date: Mon, 16 Feb 2009 01:17:14 +0000
Subject: [PATCH 05/52] 2009-02-16  Moritz  <moritz@gnu.org>

	* pam-test.c (print_help): Cosmetic change.
	(PROGRAM_VERSION): New symbol..
	(print_version): use it here.
---
 tests/ChangeLog  | 7 +++++++
 tests/pam-test.c | 9 +++++----
 2 files changed, 12 insertions(+), 4 deletions(-)

diff --git a/tests/ChangeLog b/tests/ChangeLog
index 47294a8..891aa83 100644
--- a/tests/ChangeLog
+++ b/tests/ChangeLog
@@ -1,3 +1,10 @@
+2009-02-16  Moritz  <moritz@gnu.org>
+
+	* README: Cosmetic change.
+	* pam-test.c (print_help): Cosmetic change.
+	(PROGRAM_VERSION): New symbol..
+	(print_version): use it here.
+
 2008-11-22  Moritz  <moritz@gnu.org>
 
 	* README: Improved.
diff --git a/tests/pam-test.c b/tests/pam-test.c
index 7ef30e5..9a905e3 100644
--- a/tests/pam-test.c
+++ b/tests/pam-test.c
@@ -1,5 +1,5 @@
 /* pam-test.c - simple PAM authentication test program
-   Copyright (C) 2007 g10 Code GmbH
+   Copyright (C) 2007, 2009 g10 Code GmbH
  
    This file is part of Poldi.
  
@@ -29,7 +29,8 @@
 
 
 
-#define PROGRAM_NAME "pam-test"
+#define PROGRAM_NAME    "pam-test"
+#define PROGRAM_VERSION "0.2"
 
 /* Use the standard conversation function from libpam-misc. */
 static struct pam_conv conv =
@@ -42,7 +43,7 @@ static void
 print_help (void)
 {
   printf ("\
-Usage: %s [options] <service name>\n\
+Usage: %s [options] <PAM service name>\n\
 Test PAM authentication.\n\
 \n\
 Options:\n\
@@ -56,7 +57,7 @@ Report bugs to <moritz@gnu.org>.\n", PROGRAM_NAME);
 static void
 print_version (void)
 {
-  printf ("pam-test 0.1\n");
+  printf (PROGRAM_NAME " " PROGRAM_VERSION "\n");
 }
 
 static void
-- 
2.17.0

